Assets/Scripts/Layers/LayerUIManager.cs

```diff
@@ -80,15 +81,22 @@ private void DestroyAllUIs()
         private void OnEnable()
         {
             ReconstructHierarchyUIs();
-            projectData.LayerAdded.AddListener(CreateNewUI);
-            projectData.LayerDeleted.AddListener(OnLayerDeleted);
+            ProjectData.Current.LayerAdded.AddListener(CreateNewUI);
+            ProjectData.Current.LayerDeleted.AddListener(OnLayerDeleted);
+            // projectData.OnDataChanged.AddListener(OnProjectDataChanged);
```

Comment:
Can you remove all commented and dead code before issuing a PR?

Reply:
oops, this one slipped through, will remove it